lib/nuget-parser/dotnet-core-parser.js

```diff
@@ -123,38 +121,43 @@ function getTargetObjToRun(manifest) {
   return manifest.targets[selectedTargetKey];
 }
 
+module.exports = {
+  parse: function (tree, fileContent, fileContentParser) {
```

Comment:
`async`?

Reply:
next refactor I'll make it all typescript and update to es2017